dev/lib/entities/enemy.js

```diff
@@ -159,11 +156,15 @@ class Enemy extends Entity {
       Math.random() <= game.chanceForEnemiesToGenerate &&
       numEnemies < Enemy.numCap
     ) {
-      new Enemy(gameId);
+      const enemy = new Enemy(gameId);
+      const enemySpawnPoint = Entity.getEntitySpawnPoint(enemy);
```

Comment:
Sorry to nitpick! I feel like this still breaks the modularity of the classes by having to set their constructor properties outside of the constructor (actually I think it was better before when it was at least a config getting passed in). Is it possible to assign X and Y inside the constructor?

Reply:
woah it actually is! I thought about doing it that way at first but then I thought we wouldn't be able to pass the entity props to the getSpawnPoint in the constructor because they wouldn't exist, but I guess they do. Lesson learned. pushing now